src/dev/impl/DevToys/Views/Tools/Converters/Timestamp/TimestampToolPage.xaml.cs

```diff
@@ -52,11 +52,12 @@ protected override void OnNavigatedTo(NavigationEventArgs e)
             {
                 if (long.TryParse(parameters.ClipBoardContent, out long timestamp))
                 {
-                    ViewModel.Timestamp = timestamp;
+                    ViewModel.CurrentTimestamp = timestamp;
                 }
                 else if (DateTime.TryParse(parameters.ClipBoardContent, out DateTime localDateTime))
                 {
-                    ViewModel.Timestamp = new DateTimeOffset(localDateTime.ToUniversalTime()).ToUnixTimeSeconds();
+                    // TODO:
```

Comment:
To do what? :)

Reply:
I totally missed it!
I set the breakpoints but could not reach them, so I left it at that.

In the current main branch code, what conditions go into the if?
It looks like it will have date text in the future, but right now I think it is only numerical.